init.sls

```diff
@@ -134,6 +134,15 @@ createuser-{{ index }}:
     {% endif %}
     - user: postgres
 
+{% set connection_limit = config.get('connection_limit') %}
+{% if connection_limit %}
+set_user_{{ config['username'] }}_connection_limit:
+  cmd.run:
+    - name: psql -U postgres -c "ALTER ROLE {{ config['username'] }} CONNECTION LIMIT {{ connection_limit }};"
+    - unless: psql -U postgres -c "SELECT rolconnlimit FROM pg_roles WHERE rolname = '{{ config['username'] }}';" | grep -w {{ connection_limit }}
```

Comment:
Using `-t` should be less error prone than grepping with `-w`?

Reply:
Also I think `grep -q` is a good idea

Reply:
`-t` does not seem to exist, perhaps you meant a different flag? this is an integer field, `-w` would match the exact value

Reply:
I meant `-t` for `psql`. But using `-w` sounds great then, as it prevents false positives.

Reply:
ah, got it. just updated 👍